bench-tps/src/bench.rs

```diff
@@ -544,12 +545,23 @@ fn get_nonce_blockhash<T: 'static + BenchTpsClient + Send + Sync + ?Sized>(
     client: Arc<T>,
     nonce_account_pubkey: Pubkey,
 ) -> Hash {
-    let nonce_account = client
-        .get_account(&nonce_account_pubkey)
-        .unwrap_or_else(|error| panic!("{:?}", error));
-    let nonce_data = nonce_utils::data_from_account(&nonce_account)
-        .unwrap_or_else(|error| panic!("{:?}", error));
-    nonce_data.blockhash()
+    loop {
+        match client
+            .get_account_with_commitment(&nonce_account_pubkey, CommitmentConfig::processed())
```

Comment:
Side note, if you want to speed this part up, feel free to use `get_multiple_accounts_with_commitment` and pass in chunks of nonce accounts

Reply:
This is exactly what I needed. Picking accounts one by one is super slow